src/geometry_canvas.jl

```diff
@@ -425,7 +427,7 @@ function add_events!(c::GeometryCanvas{<:Point};
 
 
     # Mouse down event
-    on(events(ax.scene).mousebutton, priority=100) do event
+    on(events(ax.scene).mousebutton, priority=-100) do event
```

Comment:
Does this fix the issue you posted about?

Reply:
It might create other issues though... like you wont be able to draw over Tyler.jl maps

Reply:
If it's just a plot then no...and it looks like you are using Consume(true) in places as well so it shouldn't be an issue there.  Why would Tyler be a problem here?

Reply:
Like shouldn't popups have a priority of `typemax(Int)` ? @SimonDanisch how does this work exactly?

Reply:
Menu has a priority of 65 ^^ so I guess it needs to below that? And all others should have 1 as a default...
Anything unimportant is negative and system stuff should have something like max int, I think.
Definitely needs some better docs

Reply:
Oh good maybe MakieDraw can be 50 then rather than 100